### PR TITLE
Call local api from claims list page

### PR DESCRIPTION
**Pull request by jbalboni**
Related to [this item](https://github.com/department-of-veterans-affairs/sunsets-team/issues/25), but I'm not closing it because it's not fully done.

This calls the local api to get claims records. It uses as much data as we can from the api, but a couple things are missing or not in the test records we have.

I've also moved over Rx's pagination component and written some unit tests for the code I added.

This is the only changed part of the UI:

screen shot 2016-10-05 at 12 40 20 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@cagedcrown Can you review this when you have a chance?

**Comment by jbalboni** [comment]
@U-DON I copied the Pagination component and styles into common here without touching Rx. I figured one of us could update Rx to use the common version in a separate PR.

**Review by molindoz — APPROVED** [review]
Everything looks good to me 👍

File: src/js/disability-benefits/components/ClaimsListItem.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default function ClaimsListItem({ claim }) {
+  const filesNeeded = claim.attributes.trackedItems.stillNeedFromYouList ? claim.attributes.trackedItems.stillNeedFromYouList.length : null;
+  return (
+    <Link className="claim-list-item" to={`/your-claims/${claim.attributes.evssId}/status`}>
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: Let's use our internal ID here, the route that you will call to get the details will use that ID too

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: I tried that and it seemed like the endpoint url was expecting the evss id.

Comment (jbalboni) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: Done

File: src/js/disability-benefits/components/ClaimsListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default function ClaimsListItem({ claim }) {
+  const filesNeeded = claim.attributes.trackedItems.stillNeedFromYouList ? claim.attributes.trackedItems.stillNeedFromYouList.length : null;
```
Comment (austin-martinez) [inline src/js/disability-benefits/components/ClaimsListItem.jsx]: Yep, sounds right

**Pull request merged**